python/pshexs/elasticsearch.py

```diff
@@ -0,0 +1,73 @@
+import elasticsearch
```

Comment:
Let's use the same directory structure here if possible, so the individual services should be in an `examples` directory.  (Or we change PHP and Python to some other name; as long as it's consistent I don't feel all that strongly about what the name is.)

Reply:
Will do - made a note in main comment.